CHANGELOG.md

```diff
@@ -1,9 +1,6 @@
-## 2.31.0-rc.1 / 2021-10-25
-
-* [BUGFIX] UI: Fix assets loading when a prefix is used in `--web.route-prefix` or `--web.external-url`. #9586
-
-## 2.31.0-rc.0 / 2021-10-21
+## 2.31.0 / 2021-11-02
 
+* [CHANGE] UI: Remove standard editor in favour of the codemirror-based editor. #9452
```

Comment:
I'd clarify what editor this is, since not everyone will guess that it's about the PromQL one (although it's pretty much the only one in the UI).